app.js

```diff
@@ -23,12 +25,24 @@ app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(cleanStrings);
 app.use('/docs', swagger.serve, swagger.setup(swaggerDocument));
+
+app.use(session({
+  secret: 'process.env.PASSPORT_SECRET',
+  resave: true,
+  saveUninitialized: true
+}));
+app.use(passport.initialize());
+app.use(passport.session());
 app.use('/api/v1', router);
 
-app.get('*', (request, response) => {
+app.get('/', (request, response) => {
```

Comment:
What happens if i make a `post` request to the '/' route?

Reply:
The / route picks up the home route while the * route picks up every route that does not exist.